implementations/ruby-json_schemer/matrix_versions.json

```diff
@@ -0,0 +1,21 @@
+[
+  "2.3.0",
+  "2.2.1",
+  "2.2.0",
+  "2.1.1",
+  "2.1.0",
+  "2.0.0",
+  "1.0.3",
+  "1.0.2",
+  "1.0.1",
+  "1.0.0",
+  "0.2.25",
+  "0.2.24",
+  "0.2.23",
+  "0.2.22",
+  "0.2.21",
+  "0.2.20",
+  "0.2.19",
+  "0.2.18",
+  "0.2.17"
+]
```

Comment:
here I've only added upto `v0.2.17` as my refactored harness code is only working upto that version. Will definitely add further more afterwards 👍.